cibuild.cmd

```diff
@@ -42,8 +42,8 @@ if not "%BuildTimeLimit%" == "" (
 ) else (
     set RunProcessWatchdog=false
 )
-    
-call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\VsDevCmd.bat" || goto :BuildFailed
+
+call "%RoslynRoot%SetDevCommandPrompt.cmd" || goto :BuildFailed
```

Comment:
I don't like this approach.  It means the build is going to silently switch to Dev15 if Dev15 is installed.  What are the options for forcing the build back to Dev14?  That is still a scenario we have to uspport.